src/components/Pagination/Pagination.jsx

```diff
@@ -56,19 +56,13 @@ class Pagination extends Component {
     this.state = { activePage: props.activePage };
   }
 
-  handleItemClick = (e, nextActivePage) => {
+  handleItemClick = (e, item) => {
```

Comment:
non sono troppo sicura di tutte queste modifiche al componente pagination, perchè funziona dalle altre parti. Oppure c'è un caso specifico in cui non funziona?

Reply:
Al momento la paginazione è su 3 oggetti, così com'è da errore se non si usa in modo complicato gestendo in due modi activepage ed è contro intuitivo. Conviene sistemare ora, anziché portarci la complessità nel futuro